### PR TITLE
MAT-9842: Update spring to resolve vulnerability

### PR DESCRIPTION
**Pull request by mcmcphillips**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-9842](https://jira.cms.gov/browse/MAT-9842)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
 		<checkstyle.file>madie-checkstyle.xml</checkstyle.file>
 		<gov.cms.madie.models.version>0.6.56-SNAPSHOT</gov.cms.madie.models.version>
 		<com.jayway.jsonpath.version>2.9.0</com.jayway.jsonpath.version>
-		<org.springframework.boot.version>3.5.12</org.springframework.boot.version>
+		<org.springframework.boot.version>3.5.13</org.springframework.boot.version>
```
Comment (sb-prateekkeerthi) [inline pom.xml:28]: we also need to update snapshot version as well so that new version would be published which then can be pulled into other services

<artifactId>madie-translator-commons</artifactId>
 <version>3.0.4-SNAPSHOT</version> ---> <version>3.0.5-SNAPSHOT</version>

**Review by sb-prateekkeerthi — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (mcmcphillips) [reply pom.xml:28]: ty

**Review by sb-prateekkeerthi — APPROVED** [review]
